execution/executor/sortition.go

```diff
@@ -22,6 +22,9 @@ func (e *SortitionExecutor) Execute(trx *tx.Tx, sb sandbox.Sandbox) error {
 	if val == nil {
 		return errors.Errorf(errors.ErrInvalidTx, "unable to retrieve validator")
 	}
+	if val.Power() == 0 {
```

Comment:
Please add this comment above this statement: "Power for parked validators are set to zero"